airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -306,6 +306,9 @@ class GKEStartPodOperator(KubernetesPodOperator):
         pod; if False, leave the pod.  Current default is False, but this will be
         changed in the next major release of this provider.
     :type is_delete_operator_pod: bool
+    :param in_cluster: Run kubernetes client with in_cluster configuration. This parameter is
+        for backward compatibility. No need to change this parameter from default now.
+    :type in_cluster: bool
```

Comment:
If the user can't really sent the value of this parameter then why should we expose it in the constructor?